coot/moorhen-wrappers.cc

```diff
@@ -191,6 +356,14 @@ class molecules_container_js : public molecules_container_t {
             return DrawSugarBlocks(mol,cid_str);
         }
 
+        std::vector<TableEntry> privateer_validate(int imol) {
+            std::cout << "[Privateer] Calculating Carbohydrate Validation for Molecule Number:  " << imol << std::endl;
```

Comment:
Please remove the debug messages, I'm trying to keep these to a minimum and we already have a lot in libcoot api.